welcome/run-d-program-locally.md

```diff
@@ -0,0 +1,40 @@
+# Eseguire programmi D sul proprio computer
+
+Il linguaggio D è accompagnato non solo dal compilatore `dmd`, ma anche dallo pseudo-interprete `rdmd` e dal gestore di pacchetti `dub`.
+
+### Il compilatore DMD
+
+Il compilatore *DMD* compila file D per produrre un file binario eseguibile.
+Sulla riga di comando *DMD* dev'essere invocato seguito dal/dai file da compilare:
```

Comment:
"Nella"? 
Anyway this doesn't sounds good "invocato seguito" seems a mistake :)
I think something like "[...] invocato specificando la lista dei file da compilare:" or something like this.